### ISSUE TITLE
Logging information

### ISSUE THREAD

**Issue by croffler** [Ninja-Squad/DbSetup#62]
We have a complex system with many data sets. When there is an error in one of the data sets we get this exception:
com.ninja_squad.dbsetup.DbSetupRuntimeException: org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block
it is difficult to find the actual problem.  As far as I can tell, there is no logging in DbSetup.  
Would be nice  to have some more information

**Comment by jnizet** [comment]
@croffler Yes, I didn't want to impose a logging framework once again, and wanted to be dependency-free. I agree exceptions can sometimes be cryptic, and I'll maybe try to add some logging mechanism.

In the meantime, can't you find the root cause based on the exception stack trace, or using your debugger?

**Comment by croffler** [comment]
Makes sense as far as the dependency-free.

We are able to debug it and find eventually the problems .... but a little
help with logging I think would be great, when you have over a hundred
data sets :)

BTW  DbSetup is really cool , keep up the good work

Chris

**Comment by softdays** [comment]
+1, it should be very helpful to trace sql query
look at slf4j that's what it's meant for

**Comment by gnostrenoff** [comment]
+1, logging would be welcome :)

**Comment by jkttt** [comment]
+1 for logging

**Comment by raderio** [comment]
+1 for logging
